src/components/DefineCompany/CooperativeType.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-form ref="cooperativeTypeForm" id="cooperative-type">
+    <v-row class="py-3" no-gutters>
+      <v-col>
+        <label class="font-weight-bold">Select Type</label>
```

Comment:
Let's be consisten with labels: either use the class, or `<strong>`. (I've gone with strong.)

Reply:
Sounds good! I don't mind strong, will update.

Reply:
Updated